Package.swift

```diff
@@ -0,0 +1,70 @@
+// swift-tools-version: 5.9
```

Comment:
Based on the description of this property on line 2 (i.e. it's "the minimum version of Swift required to build this package"), do we need such a high Swift tools version?

Reply:
This was the default one for the newest Xcode, we can go down, but how much do we want?